src/components/encounter/encounter-form.component.tsx

```diff
@@ -85,12 +80,18 @@ const EncounterForm: React.FC<EncounterFormProps> = ({
   setAllInitialValues,
   isSubmitting,
 }) => {
+  const formSessionDate = useMemo(() => new Date(), []);
   const { t } = useTranslation();
+  const session = useSession();
```

Comment:
I would load the session at the a higher level (form-engine) given that fact that it's possible to have multiple instances of the `<EncounterForm ../>` in a single form.